scalding-serialization/src/main/scala/com/twitter/scalding/serialization/Exported.scala

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2015 Twitter, Inc.
```

Comment:
can we skip the bogus copyright?

1) year is wrong
2) stripe is maybe paying you to write this code.

Reply:
Yeah i was going to ask you if you thought we need to put the licence and/or copyright in these at all